content/en/service_management/app_builder/components/custom_charts.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Custom Charts
+disable_toc: false
+aliases:
+- /service_management/app_builder/custom_charts/
+further_reading:
+- link: "/service_management/workflows/components/"
+  tag: "Documentation"
+  text: "Components"
+- link: "/service_management/workflows/build/"
+  tag: "Documentation"
+  text: "Build Apps"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">App Builder is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
+{{< /site-region >}}
+
+This page provides an example of how to use the custom chart component in your App Builder apps.
+
+The basic workflow for creating a custom chart is as follows:
+
+1. Create or choose a query that contains the data you want to graph.
+1. Add the custom chart component and choose a chart style from the App Builder UI or the [Vega-Lite example gallery][1].
+1. Replace the values in the example with your data.
+
+The following example shows how to create a histogram chart illustrating Datadog Logs grouped by service.
```

Comment:
Should we make this a header or bold? That way the 2 subheaders below are clearly part of an example

Reply:
```suggestion
## Example Setup Flow

### A histogram chart illustrating Datadog Logs grouped by service.
```

If we did something like this, the headers below might be better as `###` instead of `##`

Reply:
Applied the spirit of this suggestion :)